pkg/spdk_test.go

```diff
@@ -98,7 +91,7 @@ func startTarget(spdkDir string, args []string, execute func(envs []string, bina
 		fmt.Sprintf("%s %s", filepath.Join(spdkDir, "build/bin/spdk_tgt"), argsInStr),
 	}
 
-	_, err = execute(nil, "sh", tgtOpts, 1*time.Minute)
+	_, err = execute(nil, "sh", tgtOpts, 10*time.Minute)
```

Comment:
Should we use a no-timeout executor instead?

Reply:
Yes, we can, but 10 minutes is enough for the test.